src/components/Common/Tooltip/index.tsx

```diff
@@ -1,16 +1,51 @@
 import clsx from "clsx";
-import React from "react";
+import React, { useEffect } from "react";
 
 const Tooltip: React.FC<{
   children?: React.ReactNode;
   tooltip: React.ReactNode;
   className?: string;
-}> = ({ children, tooltip, className }) => {
+  wrapperClassName?: string;
+}> = ({ children, tooltip, className, wrapperClassName }) => {
+  const wrapperRef = React.useRef<HTMLSpanElement>(null);
+  const tooltipRef = React.useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
+    function fixPosition() {
+      const rect = wrapperRef.current?.getBoundingClientRect();
+      const tooltip = tooltipRef.current;
+
+      const tooltipRect = tooltip.getBoundingClientRect();
+
+      let left = -tooltipRect.width / 2 + 8;
```

Comment:
any reason why the magic numbers `8` and `32` are being used? may be a good idea to at least leave a comment explaining why (for better maintainability)